### PR TITLE
Adds Helium Mobile Service Provider HIP

### PR DESCRIPTION
**Pull request by meowshka**

### CODE REVIEW

**Comment by PharkMillups** [comment]
Rendered view here:

https://github.com/helium/HIP/blob/0ebe037fbc281d5fdb2eb93e7ebd9a547453e412/0000-helium-mobile-service-provider.md

File: 0000-helium-mobile-service-provider.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+
+### Onboarding to the Helium Mobile Network
+
+In compliance with HIP53 staking requirements for Service Providers, Helium Mobile will stake 500M MOBILE for the right to operate as a Service Provider on the Helium Mobile Network.
+
+Once the required amount is staked, Helium Mobile Service Provider will be minted as an NFT by the ‘Helium entity manager’ smart contract on the Solana blockchain and will become a rewardable entity.
+
+This stake will need to remain locked and will not earn any staking rewards, as long as the Service Provider remains as part of the MOBILE subDAO. In lieu of staking rewards, Service Provider has the right to claim up to 10% of MOBILE rewards (provided [HIP79](https://github.com/helium/HIP/blob/main/0079-mobile-poc-mappers-rewards.md) passes) in the Service Provider bucket. 
+
+This HIP proposes a cooldown period of 3 months before the Service Provider can claim the staked amount after it stops data offloading to the Mobile Network.
```
Comment (mawdegroot) [inline 0000-helium-mobile-service-provider.md]:
```suggestion
This HIP proposes a cooldown period of 3 months before the Service Provider can claim the staked amount after it stops data offloading to the Helium Mobile Network.
```

File: 0000-helium-mobile-service-provider.md [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+
+The 30Gb rewards limit will reset with a start of a new 30-day billing cycle for each Subscriber with an unlimited data plan. The data traffic cap is not limited to a particular Hotspot. It is applicable to all Hotspots that provide data for a specific Subscriber.
+
+It’s important to note that Helium Mobile offers an unlimited plan without data caps  for Subscribers. This means that the data offloading will still continue after the 30Gb usage is reached, however Hotspot Owners won’t earn MOBILE rewards for providing data traffic to such Subscribers. Additionally, DCs won’t be burned from the Helium Mobile Carrier account for the data traffic routing after the data caps are reached for Subscribers.
+
+We don’t anticipate data caps to be reached frequently. Nevertheless, we will closely analyze usage on the Network and quickly iterate on solutions to ensure all members of the Network can earn rewards for legitimate use of the Network.
```
Comment (mawdegroot) [inline 0000-helium-mobile-service-provider.md]:
```suggestion
We do not anticipate that the 30GB data cap will be reached frequently. Nevertheless, we will closely analyze the data usage of our Subscribers and quickly iterate on solutions in order to ensure all members of the Network will be appropriately rewarded for legitimate use of the Network.
```

**Review by vincenzospaghetti — APPROVED** [review]

**Comment by vincenzospaghetti** [comment]
@mawdegroot, your edits we're merged! Great review.

**Pull request merged**